metricbeat/module/php_fpm/_meta/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM richarvey/nginx-php-fpm:1.1.2
+FROM tsouza/nginx-php-fpm
```

Comment:
SGTM. Interestingly @richarvey did that also previously and then switched to an other versioning system. Would be interesting to know why.